3기/choijiyoung/week2/index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>CHOIJI's portfolio</title>
+
+    <!-- core -->
+    <link rel="stylesheet" href="./css/style.css" />
+    <link rel="stylesheet" href="./css/main.css" />
+  </head>
+
+  <body>
+    <div id="wrap">
+      <header id="header">
+        <h1 class="title">CHOIJI</h1>
+
+        <button class="responsive__menu"></button>
+        <nav class="nav">
+          <div class="tab-container">
+            <button data-tab-target="#tab1" class="button active" type="button">
+              ABOUT
+            </button>
+            <button data-tab-target="#tab2" class="button" type="button">
+              TIMELINE
+            </button>
+          </div>
+        </nav>
+      </header>
+
+      <main id="main">
+        <section id="tab1" class="tab-content tab1-content active">
+          <h2 class="screen-reader-text about__title">About</h2>
+          <div class="about__description">
+            <h3 class="about__description-title">I'm Frontend Developer</h3>
+            안녕하세요! 프론트엔드 개발자 최지영입니다
+          </div>
+          <div class="about__skills">
+            <img
+              src="./images/tab1-HTML5.png"
+              class="about__skills-icon"
+              alt="HTML5"
+            />
+            <img
+              src="./images/tab1-CSS3.png"
+              class="about__skills-icon"
+              alt="CSS3"
+            />
+            <img
+              src="./images/tab1-JavaScript.png"
+              class="about__skills-icon"
+              alt="JavaScript"
+            />
+            <img
+              src="./images/tab1-typescript.png"
+              class="about__skills-icon"
+              alt="typescript"
+            />
+            <img
+              src="./images/tab1-react.png"
+              class="about__skills-icon"
+              alt="react"
+            />
+          </div>
+        </section>
+
+        <section id="tab2" class="tab-content tab2-content">
+          <article class="inner">
+            <h2 class="timeline__title">
+              TIMELINE <strong>수상 및 활동</strong>
+            </h2>
+            <ol class="timeline__list"></ol>
+          </article>
+        </section>
+      </main>
+
+      <footer id="footer">
+        Copyright 2023 <strong> @Jiyoung Choi </strong>
+      </footer>
```

Comment:
푸터에도 제목이 있으면 좋을 것 같아요!
그리고 PC 버전에서는 @ 가 나오면 안되기 때문에 

```suggestion
      <footer id="footer">
        <span class="desktop">Copyright 2023 <strong>Jiyoung Choi</strong></span>
        <span class="mobile">@Jiyoung Choi</span>
      </footer>
```

이렇게 2개의 태그를 사용해서 만드는것도 방법이예요 👍🏻